scripts/metadata_comparison/metadata_comparison/comparer.py

```diff
@@ -175,8 +272,36 @@ def sum_call_times_weighted(j: JsonObject, key: AnyStr):
     for call_key in call_keys_sorted_without_prefix:
         row = []
         call_1, call_2 = [j.get('calls').get(call_key.full) for j in [json_1, json_2]]
+        # Assumes machine types are the same for both calls
```

Comment:
Wait what!?

Reply:
Yeah this has always been the philosophy in this code, only the comment is new. In practice this is true > 99.9% of the time, the only exceptions seeming to be when PAPI v1 picks an e something beta machine type that PAPI v2 would never choose.

Reply:
Ok. I'm philosophically uneasy but tactically fine with this.